iroh-rpc-client/src/network.rs

```diff
@@ -257,6 +273,36 @@ impl P2pClient {
     }
 }
 
+#[derive(Debug)]
+pub struct Lookup {
+    pub peer_id: PeerId,
+    pub listen_addrs: Vec<Multiaddr>,
+    pub observed_addrs: Vec<Multiaddr>,
+    pub protocol_version: String,
+    pub agent_version: String,
+    pub protocols: Vec<String>,
+}
+
+impl Display for Lookup {
```

Comment:
Should the logic that displays a Lookup be in `iroh-rpc-client`? I ask because this is quite a complex structure and generally I'd expect the CLI to be in charge of how this is displayed. I realize that Rust makes us define this `Display` trait here as we can't do it in the CLI, so I'm not sure what the right way to go about it - we could get rid of `Display` and write the display logic in the CLI, or introduce a newtype in the CLI just for display purposes.

Reply:
Agree about this. Display should not be the way to do cli specific formatting. Next thing you know you would have to do localization...

Newtype or just a function for formatting in cli would be better.

Reply:
Solid. I'll change this.

*Cleaning up* the output so it looks good on the cli tho, I'll leave to a follow-up.